src/std/build-script.ss

```diff
@@ -41,33 +40,16 @@
                         (def (main . args)
                           (match args
                             (["meta"]
-                             (write '("spec" "deps" "compile"))
+                             (write '("spec" "compile"))
```

Comment:
this may be breaking `gxpkg`, i think it uses the "deps" command.

Reply:
Yup, I've commented out the `"deps"` command from `gxpkg`. I've checked that nothing in `gerbil-directory` meaningfully uses it.